doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -232,6 +232,8 @@ nRF Desktop
   * The :ref:`CONFIG_DESKTOP_BLE_DONGLE_PEER_ID_INFO <config_desktop_app_options>` configuration option.
     It can be used to indicate the dongle peer identity with a dedicated event.
   * Updated the :ref:`nrf_desktop_fast_pair_app` to remove the Fast Pair advertising payload for the dongle peer.
+  * Removed Fast Pair discoverable advertising payload on Resolvable Private Address (RPA) rotation during discoverable advertising session.
+  * Enabled synchronization of the Resolvable Private Address (RPA) rotation with the advertising data update in the Fast Pair configurations using the :kconfig:option:`CONFIG_CAF_BLE_ADV_ROTATE_RPA` option.
```

Comment:
```suggestion
  * Synchronization of the Resolvable Private Address (RPA) rotation to enable with the advertising data update in the Fast Pair configurations using the :kconfig:option:`CONFIG_CAF_BLE_ADV_ROTATE_RPA` option.
```
We could move this to the Updated section.

Reply:
```suggestion
  * Synchronization between the Resolvable Private Address (RPA) rotation and the advertising data update in the Fast Pair configurations using the :kconfig:option:`CONFIG_CAF_BLE_ADV_ROTATE_RPA` option.
```